Uni_life_tracker/app/src/main/res/layout/activity_add_expenses.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
you can align the whole activity with LinearLayout (rather than constraint) just so everything's better aligned & more universe with other screen sizes

Reply:
fixed